### DISCUSSION TITLE
disabling task received logs

### DISCUSSION THREAD

**Discussion by zalts1** [celery/celery#8353] [Q&A] [answered]
Hello, Im working on a service with several workers using celery.
Im running sort of flow manager that contains lots of tasks / periodic tasks.
Im sending all logs (INFO+) to cloud logging service.
the result is that my service spams the logging service with many "task received / "task succeed" logs.
such case also results in higher expenses for the log storing.
I was trying to find a way to disable these logs
for "success" messages I was able to find out I can set trace log level to warning so it skips such messages 
`from celery.app import trace`
`trace.logger.setLevel(logging.WARNING)`
but couldn't find similar solution to task received messages
does anyone know if there's a way to disable both?
it would be even better if I could configure it for specific tasks

thanks! 🙏

**Comment by Sinkler** [comment, marked as answer]
```python
from celery.worker import strategy
strategy.logger.setLevel(WARNING)
```

**Reply by prhbrt** [reply]
To also remove the success/result logs:
```python3
from celery.worker import strategy
from celery.app import trace
strategy.logger.setLevel(logging.WARNING)
trace.logger.setLevel(logging.WARNING)
```

**Comment by JamesonNetworks** [comment]
I still see Task succeed messages in the log after adding this code.

**Reply by JamesonNetworks** [reply]
I believe this was an error on my part, I was actually able to resolve this by changing my invocation of celery to set the log level to warning and changed relevant log entries to logger.warn instead of info. I no longer see any receive/succeed messages and was able to remove the strategy changing code
